code/app/src/main/java/com/example/the_tarlords/ui/event/EventDetailsFragment.java

```diff
@@ -236,4 +247,15 @@ else if(menuItem.getItemId()==R.id.mapOptionsMenu) {
 
     }
 
+    /**
```

Comment:
Are we requesting notifications perms per event? Or are we doing a blanket permission request? If the former, have you tested whether or not disabling notifications for one event will disable them for all events, even if you had enabled notifications for previous events?

Reply:
> Are we requesting notifications perms per event? Or are we doing a blanket permission request? If the former, have you tested whether or not disabling notifications for one event will disable them for all events, even if you had enabled notifications for previous events?

It prompts the notification permissions box whenever a user opens an event detail fragment. It is a blanket permission and changes the device setting. Can be moved to main activity if it is preferred. An assumption I made was that if a user opens an event detail fragment, they may or may not have the intention to sign up to the event and just to ensure they have notification permissions set up in case they do decide to sign up.

Reply:
Okay cool, that sounds good how it is, once we have a proper sign up button then we might want to change it a bit but for now I think that it should be good to merge.